inc/snowball-config.php

```diff
@@ -170,5 +170,60 @@ function snowball_enable_front_page($query) {
 
 add_filter('template_include', 'snowball_template');
 
+function snowball_menu() {
+
+  /*
+   *  Use the add_options_page function
+   *  add_options_page( $page_title, $menu_title, $capability, $menu-slug, $function ) 
+   *
+  */
+
+  add_options_page(
+    'Snowball Settings',
+    'Snowball Settings',
+    'manage_options',
+    'snowball-slug',
+    'snowball_options_page'
+  );
+
+}
+add_action( 'admin_menu', 'snowball_menu' );
```

Comment:
Fix whitespace to in parens to keep consistent codebase. Same for the rest of the parens.